server/controllers/work.js

```diff
@@ -31,6 +32,35 @@ function getImageIndex(ctx) {
   return imageIndex;
 }
 
+function constructCreatorsString(creators) {
+  if (creators.length > 0) {
+    const creatorsString =  creators.reduce((acc, creator, index) => {
+      if (index === 0) {
+        return `${acc} ${creator.label}`;
+      } else if (index + 1 === creators.length) {
+        return `${acc} and ${creator.label}`;
+      } else {
+        return `${acc}, ${creator.label}`;
+      }
+    }, 'by');
+    return creatorsString;
+  } else {
+    return '';
+  }
+}
+
+function constructLicenseString(licenseType) {
+  const licenseInfo = getLicenseInfo(licenseType);
+  return `<a href="${licenseInfo.url}">${licenseInfo.text}</a>`;
+}
+
+function constructAttribution(singleWork, canonicalUri) {
+  const title = singleWork.title ? `'${singleWork.title}' ` : '';
+  const creators = constructCreatorsString(singleWork.creators);
+  const license = constructLicenseString(singleWork.thumbnail.license.licenseType);
+  return `<p>${title} ${creators}. Source: <a href="${canonicalUri}">Wellcome Collection</a>. ${license}</p>`;
```

Comment:
See above.

Reply:
Probably yes, but not sure how to easily do this at the moment as we pass the attribution (work.attribution) to the 'work-drawer' component in the following way:

    {% set usingThisImageData = [
      {title: 'License info', content: licenseInfo.humanReadableText},
      {title: 'Attribution', content: work.attribution}
    ] %}

    {% componentV2 'work-drawer', {id: 'work-using-image', heading: 'Using this image'}, null, {data: usingThisImageData, topDivider: true} %}

I think this just highlights a problem with the component, that needs fixing.